compiler/rustc_typeck/src/collect.rs

```diff
@@ -2730,7 +2730,7 @@ fn codegen_fn_attrs(tcx: TyCtxt<'_>, id: DefId) -> CodegenFnAttrs {
             if let Some(list) = attr.meta_item_list() {
                 if list.iter().any(|nested_meta_item| nested_meta_item.has_name(sym::no_coverage)) {
                     tcx.sess.mark_attr_used(attr);
-                    no_coverage_feature_enabled = true;
+                    no_coverage_feature_attr = Some(attr);
```

Comment:
The nightly check would need to happen here, btw. Otherwise we'd allow `#[feature(no_coverage)]` without `#[no_coverage]`.

Reply:
You're right. Thanks. 

Nevertheless, this is now outdated.